packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import ButtonGroupButton from './ButtonGroupButton';
+import './ButtonGroup.scss';
+
+const propTypes = {
+  /**
```

Comment:
Actually, I'm not strongly on the props order in `jsx` file. Just throw the concern on our doc site. Currently, we will generate the propType docs as the same order as in the source file.
<img width="1032" alt="screen shot 2017-03-30 at 2 56 35 pm" src="https://cloud.githubusercontent.com/assets/3724179/24523589/21eb0640-1559-11e7-9597-fd33bfb8face.png">

So reviewing the propTypes likes a kind of reviewing docs.

I'm also fine that we don't concern this now. We can add an enhancement to `terra-propTypes` to make final docs in desired order(this order should be consistent across all components).

+1